afm/logging.py

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright 2020 IBM Corp.
+# SPDX-License-Identifier: Apache-2.0
+#
+import logging
+
```

Comment:
The idea is to standardize the log entries and include the fybrik relevant fields as json key/value pairs.  Where are the relevant constants, such as we have in fybrik/fybrik/pkg/logging.go?  We want to log the fybrik-uuid in all the log messages as well as the other things listed in the constants in logging.go.